docs/modules/deployment/pages/upgrade/debian/up-debian.adoc

```diff
@@ -30,11 +30,18 @@ rsync -Ppav ${OPENNMS_HOME}/etc /tmp/etc.orig
 rsync -Ppav ${OPENNMS_HOME}/jetty-webapps/opennms/WEB-INF /tmp/opennms-web-inf
 ----
 
+. Update the apt packages index:
++
+[source, console]
+----
+apt update -y
+----
+
 . Upgrade {page-component-title} packages to the newest version:
 +
 [source, console]
 ----
-apt-get upgrade opennms
+apt upgrade opennms
```

Comment:
Wondering if `apt upgrade package-name` is still valid as opposed to `apt upgrade` which upgrades all packages and might not be something we want. The apt manual only shows `apt upgrade` without the package name argument. `apt install opennms` is the way to go IMO

Reply:
@ProjectJYL, is there a way to confirm that it's still valid? Does someone else want to weigh in? @uquevedo ?

Reply:
@ProjectJYL is right, we'd want to go with `apt install opennms`, doing `apt upgrade opennms` will just trigger any other packages to be updated/upgraded along with  opennms which might not be desirable.

Reply:
Thanks. I'll create a ticket to update.